### ISSUE TITLE
Seemingly useless pilot log message (http: multiple response.WriteHeader calls)

### ISSUE THREAD

**Issue by Stono** [istio/istio#12726]
Running `istioctl proxy-status`, generates this log message in pilot.  As we have a scripe which runs that every 10 seconds to generate alerts if things aren't looking right, we get quite a bit of noise in the logs:

Feels relatively useless and could be removed in order to keep the pilot logs cleaner?

```
2019-03-24T17:13:06.473694Z     info    http: multiple response.WriteHeader calls
2019-03-24T17:13:06.614748Z     info    http: multiple response.WriteHeader calls
2019-03-24T17:13:16.579845Z     info    http: multiple response.WriteHeader calls
2019-03-24T17:13:16.802154Z     info    http: multiple response.WriteHeader calls
2019-03-24T17:13:26.642621Z     info    http: multiple response.WriteHeader calls
2019-03-24T17:13:26.798664Z     info    http: multiple response.WriteHeader calls
2019-03-24T17:13:36.754352Z     info    http: multiple response.WriteHeader calls
2019-03-24T17:13:36.899914Z     info    http: multiple response.WriteHeader calls
2019-03-24T17:13:46.980517Z     info    http: multiple response.WriteHeader calls
2019-03-24T17:13:47.009655Z     info    http: multiple response.WriteHeader calls
2019-03-24T17:13:57.042234Z     info    http: multiple response.WriteHeader calls
2019-03-24T17:13:57.052023Z     info    http: multiple response.WriteHeader calls
```

**Comment by esnible** [comment]
Turning over to Howard John, who already has an alternative PR.

**Comment by howardjohn** [comment]
Should be fixed in 1.1.2.
